Components/Order/Confirmation.php

```diff
@@ -61,7 +61,13 @@ class Shopware_Plugins_Frontend_NostoTagging_Components_Order_Confirmation
      */
     public function sendOrder(Shopware\Models\Order\Order $order)
     {
-        $shop = Shopware()->Shop();
+        try {
+            $shop = Shopware()->Shop();
+        } catch (Exception $e) {
+            // Shopware throws an exception if service does not exist
```

Comment:
Please try to the store from $order (`$order->getStore`) if the Shop is not available in `Shopware()`.